docs/source/migration/1.3.mdx

```diff
@@ -1,6 +1,7 @@
 ---
-title: Migrating to 1.3
-description: Migrate to Apollo Android 1.3
+title: Migrating to Apollo Android 1.3
+description: From 1.2.x
+sidebar_title: "v1.3"
```

Comment:
Ah these edits are from before I thought we should cut these articles from v3 entirely.. I'll get them reverted